apps/docs/content/components/Avatar.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: Avatar
+status: Experimental
+figma: 'https://www.figma.com/file/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=543%3A19830&t=Be91VGjX2BANJnQc-11'
+source: https://github.com/primer/brand/tree/main/packages/react/src/Avatar/Avatar.tsx
+storybook: '/brand/storybook/?path=/story/components-Avatar--playground'
+description: Use Avatar to display a thumbnail representation of a person.
+---
+
+import {Label} from '@primer/react'
+
+```js
+import {Avatar, Stack} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+```jsx live
+<Avatar src="https://i.pravatar.cc/160?u=80" alt="A random avatar picture" />
+```
+
+### Sizes
+
+The size of the avatar can be controlled by the `size` prop.
+
+```jsx live
+<Stack
+  direction="horizontal"
+  alignItems="center"
+  justifyContent="center"
+  gap="spacious"
+  style={{height: 200}}
+>
+  <Avatar
+    size={32}
+    src="https://i.pravatar.cc/64?u=80"
+    alt="A random avatar picture at 32px"
+  />
+  <Avatar
+    size={40}
+    src="https://i.pravatar.cc/80?u=80"
+    alt="A random avatar picture at 40px"
+  />
+  <Avatar
+    size={48}
+    src="https://i.pravatar.cc/96?u=80"
+    alt="A random avatar picture at 48px"
+  />
+  <Avatar
+    size={64}
+    src="https://i.pravatar.cc/128?u=80"
+    alt="A random avatar picture at 64px"
+  />
+  <Avatar
+    size={80}
+    src="https://i.pravatar.cc/160?u=80"
+    alt="A random avatar picture at 80px"
+  />
+</Stack>
+```
+
+### Shapes
+
+The shape of the avatar can be controlled by the `shape` prop.
+
+```jsx live
+<Stack
+  direction="horizontal"
+  alignItems="center"
+  justifyContent="center"
+  gap="spacious"
+  style={{height: 200}}
+>
+  <Avatar
+    size={64}
+    shape="circle"
+    src="https://i.pravatar.cc/128?u=80"
+    alt="A circular avatar picture"
+  />
+  <Avatar
+    size={64}
+    shape="square"
+    src="https://i.pravatar.cc/128?u=80"
+    alt="A square avatar picture"
+  />
+</Stack>
+```
+
+## Component props
+
+### Avatar <Label>Required</Label>
+
+| name        | type                         | default  | required | description                       |
+| ----------- | ---------------------------- | -------- | -------- | --------------------------------- |
+| `alt`       | `string`                     |          | `true`   | The alt text of the image         |
+| `className` | `string`                     |          | `false`  | Avatar custom class               |
+| `size`      | `32`, `40`, `48`, `64`, `80` | `40`     | `false`  | The size (in pixels) of the image |
```

Comment:
Would be cool to capture the responsive object type. E.g. You could maybe link to the type def directly?

[ResponsiveObject](https://github.com/primer/brand/blob/a348157861960b4ed7c0296957b4ebc1d9c8373c/packages/react/src/Avatar/Avatar.tsx#L12)